docs/book/src/reference/versions.md

```diff
@@ -63,18 +63,20 @@ These diagrams show the relationships between components in a Cluster API releas
 
 #### Core Provider (`cluster-api-controller`)
 
-|                   | v0.3 (v1alpha3)      | v0.4 (v1alpha4) 			| v1.0 (v1beta1)      | v1.1 (v1beta1)    | v1.2 (v1beta1)       | v1.3 (v1beta1)      |
-|-------------------|----------------------|----------------------|---------------------|-------------------|----------------------|---------------------|
-| Kubernetes v1.16  | ✓                    |                      |                     |                   |                     |                     |
-| Kubernetes v1.17  | ✓                    |                   	  |                     |                   |                     |                     |
-| Kubernetes v1.18  | ✓                    | ✓ (only workload)    | ✓ (only workload)  	| ✓ (only workload) | ✓ (only workload)   | ✓ (only workload)  |
-| Kubernetes v1.19  | ✓                    | ✓                    | ✓                   | ✓                 | ✓ (only workload)   | ✓ (only workload)  |
-| Kubernetes v1.20  | ✓                    | ✓                    | ✓                   | ✓                 | ✓                   | ✓                  |
-| Kubernetes v1.21  | ✓                    | ✓                    | ✓                   | ✓                 | ✓                   | ✓                  |
-| Kubernetes v1.22  | ✓ (only workload)    | ✓                    | ✓                   | ✓                 | ✓                   | ✓                  |
-| Kubernetes v1.23* |                      | ✓                    | ✓                   | ✓                 | ✓                   | ✓                  |
-| Kubernetes v1.24  |                      |                      |                     | ✓                 | ✓                   | ✓                  |
-| Kubernetes v1.25  |                      |                      |                     |                   | ✓                   | ✓                  |
+|                   | v0.3 (v1alpha3)      | v0.4 (v1alpha4)   | v1.0 (v1beta1)    | v1.1 (v1beta1)    | v1.2 (v1beta1)    | v1.3 (v1beta1)    | v1.4 (v1beta1)    |
```

Comment:
I don't think it's a blocker from this PR, but I think we should consider removing v1.0 and v1.1 from these tables. We now have two supported releases, and users can review those versions of the book if they want the correct version support for those releases.

I think v0.3 and v0.4 are different cases as they're different API versions, but we could also consider removing those in favor of keeping their compatibility information in their versions of the book.

Reply:
Another question: should the lowest supported Kubernetes version change? CAPI supports a lot of versions that aren't being developed.

Reply:
I think the CAPI support policy has been to assume that version of Kubernetes works up until there's a signal that it doesn't, or an actual breaking change. 

There's still tests being run against these older versions of K8s too - so the community would have to make an affirmative decision to stop supporting some of these older versions IMO - possibly a good question for a community meeting. 

There's also a couple of related issues open to enforce this in webhooks (at least for KCP)  e.g. https://github.com/kubernetes-sigs/cluster-api/issues/7010. Today we don't enforce the versions of Kubernetes at all, so that would definitely be a start IMO, but it would create a situation where choosing to not support a version becomes explicit and strict i.e. someone has to make a PR that will block Kubernetes 1.16 in KCP.

Reply:
I will bring up discussions next year in January to discuss a few of our support policies were this will be included.